CLAUDE.md

```
@@ -0,0 +1,100 @@
+```bash
+make docker.build     # Build Docker image
+make docker.test      # Test Docker image
+make release          # Build production release
+docker-compose up -d  # Run locally with Docker
+```
+
+## High-Level Architecture
+
+The application follows a domain-driven design with these core components:
+
+### Data Flow
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)